docs/integrations/long-running.md

```diff
@@ -26,6 +26,28 @@ You can distinguish it from the rest of the containers by its name:
 
 ![image](/doc_imgs/integrations/66122754-2d337b00-e5e9-11e9-8775-562e228e3fe6.png)
 
+Certain integrations are long-running by default, they do not include the "Long Running Instance" checkbox. Examples of such integrations include:
+
+- Generic Export Indicators Service
+- Generic Webhook
+- AWS SNS Listener
+- TAXII2 Server
+- TAXIIServer
+
+In these cases, the long-running behavior is inherent and cannot be modified.
+These integrations can operate in either "single engine" mode or "no engine" mode.
+
+**Single Engine:** 
+1. Select your engine in the integration configuration.
+2. When the engine is selected, a new field called "Listen Port" will appear.
+3. Specify the port number that the instance will use.
+
+**No Engine:**
+1. Select "no engine" in the integration configuration.
+2. Click **Save** and the server automatically assigns a port for the instance.
```

Comment:
So this behavior is for the 8.8 server version right?
In 8.9, it is no longer the case; they can just test regularly.
So let's add a disclaimer for this, regarding the server versions